src/signal_scope/mainwindow.ui

```diff
@@ -27,10 +27,10 @@
     <property name="title">
      <string>File</string>
     </property>
-    <addaction name="ActionOpen"/>
+    <!-- <addaction name="ActionOpen"/> -->
```

Comment:
Yes I messed up the branches. I started to write you an email about this but then decided to wait and do this first. I thought I had removed it.